lib/Cake/Network/CakeResponse.php

```diff
@@ -1336,7 +1336,7 @@ public function file($path, $options = array()) {
 			'download' => null
 		);
 
-		if (strpos($path, '..') !== false) {
+		if (strpos($path, '../') !== false || strpos($path, '..\\') !== false) {
```

Comment:
Yes they shouldn't be, but you can't stop devs from using '/' in paths even on windows. This check should try to project against all possible paths, whether they fail somewhere is the devs problem.